Packs/CortexAttackSurfaceManagement/Playbooks/Cortex_ASM_-_Azure_Enrichment.yml

```diff
@@ -1421,33 +1266,155 @@ tasks:
         simple: n/a
     separatecontext: false
     continueonerrortype: ""
-    view: |-
-      {
-        "position": {
-          "x": 240,
-          "y": 580
-        }
-      }
+    view: "{\n  \"position\": {\n    \"x\": 260,\n    \"y\": 530\n  }\n}"
     note: false
     timertriggers: []
     ignoreworker: false
     skipunavailable: false
     quietmode: 0
     isoversize: false
     isautoswitchedtoquietmode: false
-view: "{\n  \"linkLabelsPosition\": {\n    \"25_19_yes\": 0.6,\n    \"3_42_#default#\": 0.32,\n    \"3_9_yes\": 0.5,\n    \"4_11_yes\": 0.43,\n    \"4_42_#default#\": 0.31,\n    \"7_18_yes\": 0.5\n  },\n  \"paper\": {\n    \"dimensions\": {\n      \"height\": 2965,\n      \"width\": 2262.5,\n      \"x\": -440,\n      \"y\": -320\n    }\n  }\n}"
-inputs:
-- key: RemoteIP
```

Comment:
This key was removed intentionally?
If so, please add BC notes as customers might use that input

Reply:
Yes, but I've gone ahead and added back based on your other comment. b8c5553